ipsec_accel/ipsec_accel.py

```diff
@@ -0,0 +1,1103 @@
+sleep 2
+
+"""
+            print("i {}".format(i))
+            if i <= vf_list_len-2:
+                print("vm id {}".format(vm_id))
+                ovs_vxlan += f"""echo ""
+echo "Host IDPF Interface {vf_list[i]} MAC ({vf['mac']})  VSI ({vf['vsi_id']}:{vf['vsi_num']}:{vf['port']}) maps to"
+echo "ACC Port {acc_pr_list[i]} MAC ({acc_pr['mac']})  VSI ({acc_pr['vsi_id']}:{acc_pr['vsi_num']}:{acc_pr['port']})"
+echo "ACC Port {acc_pr_list[i]} is added to OVS bridge br{vm_id}"
+ip link del TEP{vm_id}
+ovs-vsctl del-br br-int-{vm_id}
+ip link add dev TEP{vm_id} type dummy
+ovs-vsctl add-br br-int-{vm_id}
+ovs-vsctl add-port br-int-{vm_id} {acc_pr_list[i]}
+ovs-vsctl add-port br-int-{vm_id} {acc_pr_list[vf_list_len-1]}
+ifconfig TEP{vm_id} {local_vtep[vm_id]}/24 up
+ovs-vsctl add-port br-int-{vm_id} vxlan{vm_id} -- set interface vxlan{vm_id} type=vxlan options:local_ip={local_vtep[vm_id]} options:remote_ip={remote_vtep[vm_id]} options:key=1{vm_id} options:dst_port=4789
+ifconfig br-int-{vm_id} up
+"""
+
+
+            vf_to_vm += f'''
+echo ""
+ip addr add {vm_ip_list[vm_id]}/24 dev {vf_list[i]}
+ifconfig {vf_list[i]} up
+sleep 2
+
+'''
+
+            vm_id += 1
+
+        mac_list.append(str(acc_pr['mac']))
+
+
+    misc = """p4rt-ctl add-entry br0 linux_networking_control.ipv4_lpm_root_lut "user_meta.cmeta.bit16_zeros=4/65535,priority=2048,action=linux_networking_control.ipv4_lpm_root_lut_action(0)"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=0/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=1/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=2/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=3/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=4/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=5/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=6/7,priority=1,action=linux_networking_control.bypass"
+p4rt-ctl add-entry br0 linux_networking_control.tx_lag_table "user_meta.cmeta.lag_group_id=0/255,hash=7/7,priority=1,action=linux_networking_control.bypass"
+"""
+
+    acc_path = test_setup.test_config['test_params']['acc_path']
+    acc_p4_path = f'{acc_path}/fxp-net_linux-networking'
+    file = f'{path}/es2k_skip_p4.conf'
+    p4_config = 'cat <<EOF > ./'+file+'''
```

Comment:
Suggestion for future improvement: Use a file and ingest that instead of hardcoding values inside a Python script